docs/text.md

```diff
@@ -9,7 +9,61 @@ A React component for displaying text.
 
 In the following example, the nested title and body text will inherit the `fontFamily` from `styles.baseText`, but the title provides its own additional styles. The title and body will stack on top of each other on account of the literal newlines:
```

Comment:
@Simek @rachelnabors @tsawan 

Do you think that here we can have examples of how to use each props?

Reply:
Re:
> Do you think that here we can have examples of how to use each props?

I'd love to see that in the main example! Is it possible, @tsawan? Perhaps in a follow up submit?